tests/test_news_flash.py

```diff
@@ -268,3 +317,49 @@ def test_classification_statistics_ynet():
     assert precision > BEST_PRECISION_YNET
     assert recall > BEST_RECALL_YNET
     assert f1 > BEST_F1_YNET
+
+
+@contextmanager
+def _managed_waze_accident_alert():
+    waze_alert = _create_waze_accident_alert()
+    try:
+        yield waze_alert
+    finally:
+        _delete_waze_alert(waze_alert.id)
+
+
+def _create_waze_accident_alert():
```

Comment:
Now that you're using the context manager, there's no need to write these two functions separately. Just inline them.